pimd/pim6_cmd.c

```diff
@@ -430,8 +430,20 @@ DEFPY (ipv6_pim_rp,
        "Group Address range to cover\n")
 {
 	const char *group_str = (gp_str) ? gp_str : "FF00::0/8";
+	int ret;
```

Comment:
yeah this should be fixed in the cli parser not here and especially not with str2prefix and then prefix2str

tolower or toupper needs to be added in the cli somewhere so it can be normalized by the cli across all of frr